sidecar/src/figwheel_sidecar/system.clj

```diff
@@ -130,7 +131,13 @@
                    channel-id msg-data callback))
   (-connection-data [this]
     (server/-connection-data (:figwheel-server @system)))
-  (-actual [this] (:figwheel-server @system)))
+  (-actual [this] (:figwheel-server @system))
+  Suspendable
+  (suspend [c] c)
+  (resume [c old-c]
```

Comment:
Yeah, that's right, ideally `resume` and `suspend` are idempotent, in the same way that `start` and `stop` are.